packages/core/src/components/dropdown/dropdown-vars.scss

```diff
@@ -10,6 +10,9 @@
   --tds-dropdown-helper-color: var(--tds-grey-700);
   --tds-dropdown-placeholder-color: var(--tds-grey-700);
   --tds-dropdown-disabled-color: var(--tds-grey-400);
+  --tds-dropdown-scrollbar-background: var(--tds-grey-100);
```

Comment:
This is not used?

Reply:
Refactoring error, I changed variable name from --tds-dropdown-scrollbar-background to --tds-dropdown-scrollbar-track-color. Fix committed.